ci-scripts/gen_api_server.sh

```diff
@@ -3,6 +3,8 @@
 # Script to generate api-server crate from OpenAPI definition.
 # Requires augeas/augtool to be installed
 
+set -e
+
 DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
 cd $DIR/..
```

Comment:
https://www.shellcheck.net/
```bash
$ shellcheck myscript
 
Line 9:
cd $DIR/..
   ^-- [SC2086](https://www.shellcheck.net/wiki/SC2086) (info): Double quote to prevent globbing and word splitting.

Did you mean:
cd "$DIR"/..
```